### PR TITLE
fix #1996, EmbeddedCluster requires master configuration like ClusterActorRefProvider

### LINKED ISSUE #1996: WordCount cannot run with LocalCluster
LocalCluster fails with 

```
[OneForOneStrategy] ActorSystem [akka://master] needs to have a 'ClusterActorRefProvider' enabled in the configuration, currently uses [akka.remote.RemoteActorRefProvider]
akka.actor.ActorInitializationException: exception during creation
        at akka.actor.ActorInitializationException$.apply(Actor.scala:166)
        at akka.actor.ActorCell.create(ActorCell.scala:596)
        at akka.actor.ActorCell.invokeAll$1(ActorCell.scala:456)
        at akka.actor.ActorCell.systemInvoke(ActorCell.scala:478)
        at akka.dispatch.Mailbox.processAllSystemMessages(Mailbox.scala:263)
        at akka.dispatch.Mailbox.run(Mailbox.scala:219)
        at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:397)
        at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
        at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
        at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
        at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
Caused by: akka.ConfigurationException: ActorSystem [akka://master] needs to have a 'ClusterActorRefProvider' enabled in the configuration, currently uses [akka.remote.RemoteActorRefProvider]
        at akka.cluster.Cluster.<init>(Cluster.scala:79)
        at akka.cluster.Cluster$.createExtension(Cluster.scala:42)
        at akka.cluster.Cluster$.createExtension(Cluster.scala:37)
        at akka.actor.ActorSystemImpl.registerExtension(ActorSystem.scala:713)
        at akka.actor.ExtensionId$class.apply(Extension.scala:79)
        at akka.cluster.Cluster$.apply(Cluster.scala:37)
        at io.gearpump.cluster.master.ClusterReplication$class.$init$(ClusterReplication.scala:43)
        at io.gearpump.cluster.master.InMemoryKVService.<init>(InMemoryKVService.scala:31)
        at io.gearpump.cluster.master.Master$$anonfun$1.apply(Master.scala:58)
        at io.gearpump.cluster.master.Master$$anonfun$1.apply(Master.scala:58)
        at akka.actor.TypedCreatorFunctionConsumer.produce(Props.scala:343)
        at akka.actor.Props.newActor(Props.scala:252)
        at akka.actor.ActorCell.newActor(ActorCell.scala:552)
        at akka.actor.ActorCell.create(ActorCell.scala:578)
        ... 9 more
```

### PR DESCRIPTION
**Pull request by clockfly**

### CODE REVIEW

**Comment by clockfly** [comment]
To test the EmbededCluster, you can try 
```
bin\gear app -jar examples\wordcount-{version-assembly.jar -debug true -sleep 60
```

After building the package with "sbt pack"

File: integrationtest/core/src/it/scala/io/gearpump/integrationtest/checklist/CommandLineSpec.scala [hunk 1]
```diff
@@ -110,7 +118,7 @@ class CommandLineSpec extends TestSpecBase {
     commandLineClient.listApps().length + 1
   }
 
-  private def expectSubmitAppSuccess(jar: String): Int = {
+  private def expectSubmitAppSuccess(jar: String, args: String = ""): Int = {
```
Comment (huafengw) [inline integrationtest/core/src/it/scala/io/gearpump/integrationtest/checklist/CommandLineSpec.scala]: Seems the args are not used

**Comment by huafengw** [comment]
Are the UT failure irrelevant?

**Comment by huafengw** [comment]
+1

**Pull request merged**